docs/advanced-topics/README.md

```diff
@@ -0,0 +1,3 @@
+<script>
```

Comment:
It'd be nice to eventually get a real landing page for these. That there isn't might reflect that another structure would be better.